sdk/provisioning/Azure.Provisioning/src/IBicepValue.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Provisioning.Expressions;
+using Azure.Provisioning.Primitives;
+
+namespace Azure.Provisioning;
+
+/// <summary>
+/// Represents the value of a property that could be a literal .NET value, a
+/// Bicep expression, or it could be unset.
+/// </summary>
+public interface IBicepValue
```

Comment:
nit: The docs are inconsistent with language like "Gets/Get/Tracks..." Should we be consistent with "Gets..." or "Gets or sets.."?